core-plugins/docs/Joiner-batchjoiner.md

```diff
@@ -74,6 +74,23 @@ Setting it to true can cause a large drop in performance if there are a lot of n
 **Number of Partitions:** Number of partitions to use when grouping fields. If not specified, the execution
 framework will decide on the number to use.
 
+**Distribution:** Enabling distribution will increase the level of parallelism when joining skewed data. A 
+skewed join is when multiple records from one stage (the skewed stage) join to a single record in
```

Comment:
skew is a little different than this. It is possible to have multiple records joining to a single record and still be unskewed. I think it's better to say something like:

A skewed join happens when a significant percentage of input records have the same key.

Reply:
done